README.md

```diff
@@ -122,23 +161,7 @@ If you want to opt-out from such behavior, you need to force your framework to u
 
 ### Generating patches based on local changes
 
-Palindrom automatically observes local changes. This is implemented by dirty checking, triggered in event listeners for typical browser events (`mousedown`, `mouseup`, etc). It is done by the JSON-Patch library ([source](https://github.com/Starcounter-Jack/JSON-Patch/blob/master/src/json-patch-duplex.ts#L352-L354)).
-
-To generate patches for changes made in code, you need to either simulate a browser event (recommended):
-
-```js
-var clickEvent = document.createEvent('MouseEvents');
-clickEvent.initEvent("mouseup", true, true);
-window.dispatchEvent(clickEvent);
-```
-
-Or use a low level API exposed by the JSON-Patch library, provided that you have a reference the Palindrom instance:
-
-```js
-jsonpatch.generate(palindrom.observer);
-```
-
-Future versions of Palindrom may contain a high level API for generating patches. Please follow the issue [#29](https://github.com/Palindrom/Palindrom/issues/29) to know more.
+Palindrom automatically observes local changes. This is implemented by [ES6 Proxies](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy), triggered with every modification the `palindrom.obj` object.
```

Comment:
I'm not a native English speaker, but shouldn't it be "modification of the" an "implemented with"?